docs/install.rst

```diff
@@ -51,9 +51,6 @@ Astropy also depends on other packages for optional features:
 - `bleach <https://bleach.readthedocs.io/>`_: Used to sanitize text when
   disabling HTML escaping in the :class:`~astropy.table.Table` HTML writer.
 
-- `mock <https://github.com/testing-cabal/mock>`_ (python <= 3.2) or `unittest.mock <https://docs.python.org/dev/library/unittest.mock.html>`_ (python > 3.3):
```

Comment:
I already removed this in #6566 . Since that one has been reviewed earlier and is very specific to docs, I recommend you drop this change and defer to that PR instead.

Reply:
Ok, I am happy rebase this once #6566 is merged.